tools/integrations/manage.sh

```diff
@@ -30,16 +30,15 @@ cmd_build() {
   [ -d "$path" ] || error "Path must be the root path of the integration"
 
   local run_tests=$1; shift || run_tests=true
-  # TODO re-enable build cache if needed (https://github.com/airbytehq/airbyte/issues/4508)
   echo "Building $path"
-  ./gradlew --no-daemon -no-build-cache "$(_to_gradle_path "$path" clean)"
-  ./gradlew --no-daemon -no-build-cache "$(_to_gradle_path "$path" build)"
+  ./gradlew --no-daemon "$(_to_gradle_path "$path" clean)"
```

Comment:
This doesn't affect anything, since the connectors test/publish builds aren't using any caching and are always started on a new instance. I feel that putting this here is more confusing than not so I'm removing it.